bin/propolis-server/src/lib/stats.rs

```diff
@@ -194,6 +184,7 @@ pub async fn register_server_metrics(
     virtual_machine: VirtualMachine,
     log: &Logger,
 ) -> anyhow::Result<ServerStatsOuter> {
+    // TODO(ben): Fetch sled-agent identifiers here, or before and pass them in.
```

Comment:
More for my edification than anything: the sled agent IDs end up in the `VirtualMachine`, correct? What else is there to plumb here? (The caller of this function has an `&InstanceProperties` already, so it should be able to furnish whatever IDs are missing here.)

Reply:
Oh this was an outdated comment, thanks for catching that. I was originally going to have Propolis call back to the sled-agent for these, but it's much simpler to just pass them in the API request as I ended up with here. Will remove.

Reply:
Removed in 4e6ebde